<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assembly_Calculator</class>
 <widget class="QMainWindow" name="Assembly_Calculator">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>520</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Assembly_Calculator</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QWindget{

	background-color: white; 
	border-radius: 20px;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>512</width>
     <height>520</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>512</width>
     <height>520</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #1c2b3a;
    border-radius: 20px;
}
</string>
   </property>
   <widget class="QTextEdit" name="Fundo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>780</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="seno">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cosseno">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QLabel" name="Titulo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

	color: white;
	font-size: 35px;
	font-family: &quot;Orbitron&quot;, sans-serif;
	 font-weight: bold;


}</string>
    </property>
    <property name="text">
     <string>Calculadora</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="expressao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>109</y>
      <width>490</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{

	background-color: #6633ff;

}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tangente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
	 width: 80px;
    height: 60px;
}

QPushButton:hover {
    background-color: darkgray;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #34495e;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="soma">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4B5870;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subtracao">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4B5870;    /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplicacao">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4B5870;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divisao">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4B5870;    /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exponenciacao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>x^y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quadrado">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>x^2</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="igual">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>71</width>
      <height>151</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #6633ff;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="modos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{

	background-color: #a68cf2;

}</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{

	background-color: #a68cf2;
	font-size: 15px;
	color: white;
	font-family: &quot;Orbitron&quot;, sans-serif;
	 font-weight: bold;

}</string>
    </property>
    <property name="text">
     <string>Graus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{

	background-color: #a68cf2;
	font-size: 15px;
	color: white;
	font-family: &quot;Orbitron&quot;, sans-serif;
	 font-weight: bold;

}</string>
    </property>
    <property name="text">
     <string>Radianos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parentese_1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;    /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
	 width: 80px;
    height: 60px;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parentese_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
	 width: 80px;
    height: 60px;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="raiz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>sqrt[y]{x}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="virgula">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #7A7FA0;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sinal">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;      /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{

	background-color: #a68cf2;
	font-size: 15px;
	color: white;
	font-family: &quot;Orbitron&quot;, sans-serif;
	 font-weight: bold;

}</string>
    </property>
    <property name="text">
     <string>Inversa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apagar">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>240</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FF5F7E;    /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 10px;
    padding: 10px 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pi">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #586B88;     /* Cinza azulado */
    color: #ecf0f1;                /* Texto claro */
    border: none;
    border-radius: 12px;
    padding: 10px;
    font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>PI</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
